lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -193,12 +257,28 @@ def __init__(
         num_windows : int, optional
             Number of windows to average for the uncalibrated force. Using a larger number of
             windows potentially increases the bleed, but may be useful when the SNR is low.
+        hydrodynamically_correct : bool, optional
+            Enable hydrodynamically correct model.
+        distance_to_surface : float, optional
+            Distance from bead center to the surface [m]
+            Only used when using hydrodynamically correct model. None uses the approximation valid
+            for deep in bulk.
```

Comment:
What do you think about adding an assertion to avoid having `distance_to_surface < bead_radius`?

Reply:
Yeah, that's a good point. I've added it as an assertion (+ tests).